src/App.js

```diff
@@ -1,13 +1,39 @@
 import React, { Component } from 'react'
-import { Instructions } from './components'
+import { Instructions, Counter } from './components'
 import './styles/app.css'
 
 class App extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      initialCount: 0
+    }
+
+    this.handleChange = this.handleChange.bind(this)
+    this.handleSubmit = this.handleSubmit.bind(this)
```

Comment:
You can use autobinding instead of writing these two lines of code. To autobind you need to change your function signature to something like this.
handleChange = e => {/**CODE**/}